server/middleware/validation.js

```diff
@@ -16,7 +16,10 @@ const validationErrors = {
     isValidWithinDate: label => `${label} must be within the last ${VALID_DAYS_RANGE} days`,
     validCaseReference: () => 'Case reference is not valid',
     contributionsFulfilled: () => 'Case contributions have to be completed or cancelled',
-    oneOf: () => 'Options are not valid'
+    oneOf: () => 'Options are not valid',
```

Comment:
You shouldn't use "not valid" here. If this is an option group and the problem is that they didn't pick an option, use "Select a [whatever it is]". See: https://design-system.service.gov.uk/components/radios/#error-messages